pkg/proc/bininfo.go

```diff
@@ -505,20 +505,20 @@ type Function struct {
 	rangeParentNameCache int // see rangeParentName
 	// extraCache contains informations about this function that is only needed for
 	// some operations and is expensive to compute or store for every function.
-	extraCache *functionExtra
+	extraCache *FunctionExtra
 }
 
-type functionExtra struct {
-	// closureStructType is the cached struct type for closures for this function
-	closureStructType *godwarf.StructType
+type FunctionExtra struct {
```

Comment:
Because parsing the closure type separately from the dwarf requires accessing private functions or variables of the `Image` type, such as `getDwarfTree`, `dwarf`, `index`, etc., it is not easy to obtain them.

Reply:
@aarzilli Any suggestions? Decoding dwarf externally seems quite complex.

Reply:
You can figure out which image it is by looking at Entry, the only other thing you need at that point is `offset`, which we can make public.